app/models/binary/explore/NgffExplorer.scala

```diff
@@ -48,6 +52,7 @@ class NgffExplorer(implicit val ec: ExecutionContext) extends RemoteLayerExplore
                                        remotePath: VaultPath,
                                        credentialId: Option[String],
                                        channelCount: Int,
+                                       channelAttributes: Option[Seq[(Option[Color], Option[String])]] = None,
```

Comment:
Could this be Option[ChannelAttributes] with a custom case class? I think we may get confused in the future this tuple structure here